docs/components/example.css

```diff
@@ -66,6 +66,10 @@ body.cauldron--theme-dark {
   position: relative;
 }
 
+.Component__example__react .Link--lowercase {
+  margin: 0 var(--space-smaller);
```

Comment:
Added this in so there would be space between the examples where buttons are adjacent to a link. It was looking crowded without this. 

<img width="783" alt="CleanShot 2023-05-05 at 01 35 32@2x" src="https://user-images.githubusercontent.com/1016306/236384198-a46ea7f1-183a-4e51-ab7c-7279fa037e69.png">